test/integration/client-side-encryption/driver.test.js

```diff
@@ -53,6 +53,57 @@ describe('Client Side Encryption Functional', function () {
     }
   });
 
+  describe('Collection', function () {
```

Comment:
Test LGTM.
Do we want to fix the MongoServerError from making this mistake in general though? [This bit](https://github.com/mongodb/node-mongodb-native/blob/main/src/error.ts#L181-L184) will always crash if a name is passed to the constructor

Reply:
It's not just `MongoServerError` that causes this error but `MongoBulkWriteError` as well: https://github.com/mongodb/node-mongodb-native/blob/1079167f5a61b7f10260a4d0b86bddb10087257c/src/bulk/common.ts#L731 . So to fix this I just decided that the simplest way was to make `MongoCryptError` fall in line with all our other errors and define the name getter.

Reply:
Yea let's figure this out in a follow up: NODE-4362